samcli/cli/cli_config_file.py

```diff
@@ -0,0 +1,152 @@
+"""
+CLI configuration decorator to use TOML configuration files for click commands.
+"""
+
+## This section contains code copied and modified from [click_config_file][https://github.com/phha/click_config_file/blob/master/click_config_file.py]
+## SPDX-License-Identifier: MIT
+
+import functools
+import os
+import logging
+
+import click
+import toml
+
+__all__ = ("TomlProvider", "configuration_option")
+
+LOG = logging.getLogger(__name__)
+DEFAULT_CONFIG_FILE_NAME = "sam-config"
```

Comment:
i guess this will be changed to `samconfig.toml`

Reply:
changed.